vendor/github.com/HewlettPackard/oneview-golang/ov/enclosure.go

```diff
@@ -290,7 +290,7 @@ type EnclosureCreateMap struct {
 	ForceInstallFirmware bool          `json:"forceInstallFirmware,omitempty"`
 	FirmwareBaselineUri  string        `json:"firmwareBaselineUri,omitempty"`
 	Force                bool          `json:"force,omitempty"`
-	InitialScopeUris     []string      `json:"initialScopeUris"`
+	InitialScopeUris     []utils.Nstring      `json:"initialScopeUris"`
```

Comment:
this updates are not updated to Golang SDK, please raise PR on Golang SDK as well to avoid future conflicts.

Reply:
done